Makefile

```diff
@@ -1,5 +1,13 @@
 build:
+	chmod +x ./scripts/generate_wallets.sh
+	chmod +x ./scripts/calculate_marketcap.sh
```

Comment:
This isn't necessary, as long as the scripts are committed with correct permissions.

Reply:
Removed them. Good ideas throughout.